src/classes/yaml-utils.ts

```diff
@@ -147,14 +209,9 @@ export default class YamlUtils {
       file.spec.template.spec.containers[0] = containerObj;
     }
     
-    if (args.env && file.spec.template.spec.containers[1]) {
-      for (const env in args.env) {
-        file.spec.template.spec.containers[1].env.push(env);
-      }
-    }
     if (args.replicas) {
       file.spec.replicas = args.replicas;
     }
     return file;
   }
-}
+}
```

Comment:
File starting to be quite large and confusing, would recommend putting wholes into functions and maybe even into their own file.

Reply:
Creating project utils is a matter of another issue.